src/deepsparse/pipeline.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         self.schedulers = schedulers
         self.pipeline_state = pipeline_state
         self._continuous_batching_scheduler = continuous_batching_scheduler
+        self.timer_manager = TimerManager()
```

Comment:
we should ideally make timer manager and timings optional

Reply:
additionally, as discussed before we should follow up with the default ability to not keep a history of timings